frontend/src/components/QuickAdd.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export const QuickAdd = () => {
+  return (
+    <div className="row">
+      <h2>Quick add:</h2>
+      <input
```

Comment:
Every input should have a label for accessibility reasons. We don't need to show it though (it's not part of the mockup). But something like this before the input: 
```
      <label htmlFor="issue-number" style={{ display: "none" }}>
        Type issue number
      </label>
```
If you use this, you will have to add `id="issue-number"` to the input tag.